projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -456,6 +466,43 @@ export class DatagridComponent<R> implements OnInit, AfterViewInit, OnDestroy {
         return this.contextualActionPosition && this.contextualActionPosition === ContextualActionPosition.ROW;
     }
 
+    /**
+     * The {@link maxFeaturedActionsOnRow} value depends in the contextual featured actions config which belongs to the
```

Comment:
If you're pointing to a property of the current class, the correct syntax is `{@link #maxFeaturedActionsOnRow}`